src/containers/WrappedIDO/WrappedIDO.tsx

```diff
@@ -0,0 +1,11 @@
+import { IDO } from '@components/IDO/IDO'
```

Comment:
adding container and page wasn't in scope of task, just IDO component and storybook for it